frontend/service/oauth.service.js

```diff
@@ -41,6 +44,15 @@ export class OAuthService {
       const accessToken = response.accessToken
       const username = await this.getUsername(accessToken)
       this.$store.commit('auth/login', { username, accessToken })
+      const oauth = {
+        username,
+        accessToken
+      }
+      const token = jwt.sign(oauth, 'password')
+      this.$cookies.set('oauth', token, {
+        path: '/',
+        maxAge: 60 * 60 * 24 * 7
```

Comment:
Currently accessToken has a life of just 1 hour: https://github.com/PhiloBiblon/philobiblon-ui/blob/master/backend/src/main/java/io/github/philobiblon/backend/service/impl/WikibaseOAuthServiceImpl.java#L40
I'm not sure if it will happen after an hour, maybe some error will appear.